src/main.js

```diff
@@ -550,6 +550,17 @@ async function mkdirpIgnoreError(dir) {
 
 //#region NLS Support
 
+function processZhLocale(appLocale) {
+	if (appLocale.startsWith('zh')) {
+		const region = appLocale.split('-')[1];
+		if (['hans', 'cn', 'sg', 'my'].includes(region)) {
```

Comment:
A comment would be good here so we don't need to dig through commit history to understand it